docroot/profiles/custom/cgov_site/config/install/features.bundle.cgov.yml

```diff
@@ -0,0 +1,104 @@
+uuid: abeaadad-9beb-4af4-8e29-76a372e2a1ba
```

Comment:
Custom bundle for cgov. Changes some default settings to allow core config to be exported/overridden by our modules. Enables new custom FeaturesAssignment for cgov. (See files below)